cmd/controller-manager/app/options/options.go

```diff
@@ -185,7 +185,7 @@ func (o *Options) AddFlags(flags *pflag.FlagSet, allControllers, disabledByDefau
 		"<group> for skip resources with a specific API group(e.g. networking.k8s.io),\n"+
 		"<group>/<version> for skip resources with a specific API version(e.g. networking.k8s.io/v1beta1),\n"+
 		"<group>/<version>/<kind>,<kind> for skip one or more specific resource(e.g. networking.k8s.io/v1beta1/Ingress,IngressClass) where the kinds are case-insensitive.")
-	flags.StringSliceVar(&o.SkippedPropagatingNamespaces, "skipped-propagating-namespaces", []string{},
+	flags.StringSliceVar(&o.SkippedPropagatingNamespaces, "skipped-propagating-namespaces", []string{"kube-.*"},
```

Comment:
We should update the usage too...

By the way, should we put `karmada-system/karmada-cluster/karmada-es-*` to the default list?

Reply:
> We should update the usage too...



Your meaning is that `Comma-separated namespaces that should be skipped from propagating in addition to the default skipped namespaces(karmada-system, karmada-cluster, namespaces prefixed by kube- and karmada-es-).` needs to be updated, right?

Reply:
> By the way, should we put karmada-system/karmada-cluster/karmada-es-* to the default list?



These are Karmada's reserved namespaces and users should not be allowed to create resources arbitrarily. This can protect Karmada.

Reply:
Yes.